CMakeLists.txt

```diff
@@ -443,6 +465,8 @@ if(LIBSRTP_TEST_APPS)
             ${ENABLE_WARNINGS_AS_ERRORS})
     if(ENABLE_OPENSSL)
       target_include_directories(test_srtp PRIVATE ${OPENSSL_INCLUDE_DIR})
+    elseif(ENABLE_OPENSSL)
```

Comment:
Shouldn't this be "elseif(ENABLE_WOLFSS)" ?

Reply:
Yes, it should have been!
Fixed and removed the change to to macOS builds to force /usr/local/include.